doc/for-developers/vcard_doc.md

```diff
@@ -0,0 +1,27 @@
+# Import vcard
+
+We can import one or more vcards from a file.
+
+## Import vcard
+
+```
+cd src/backend
+caliopen import_vcard --help
+caliopen -f configs/caliopen.yaml.template import_vcard -u <username> [-d <directory> | -f <file>]
```

Comment:
shall give the equivalent command using cli in docker container (docker-compose run cli ....)